README.md

```diff
@@ -23,25 +23,129 @@ import {
 } from 'react-native';
 import InfiniteScrollView from 'react-native-infinite-scroll-view';
 
+class ExampleComponent extends React.Component {
+  async _loadMoreContentAsync() {
+    // Fetch more data here.
+  }
 
-// Inside of a component's render() method:
-render() {
-  return (
-    <ListView
-      renderScrollComponent={props => <InfiniteScrollView {...props} />}
-      dataSource={...}
-      renderRow={...}
-      canLoadMore={this.state.canLoadMoreContent}
-      onLoadMoreAsync={this._loadMoreContentAsync()}
-    />
-  );
+  render() {
+    return (
+      <ListView
+        renderScrollComponent={props => <InfiniteScrollView {...props} />}
+        dataSource={...}
+        renderRow={...}
+        canLoadMore={this.state.canLoadMoreContent}
+        onLoadMoreAsync={this._loadMoreContentAsync.bind(this)}
+      />
+    );
+  }
 }
 ```
 
+A more complete example that uses a `ListView.DataSource`, [react-redux](https://github.com/reactjs/react-redux), and supports pagination would look something like this:
+
+```js
+import React from 'react';
+import {
+  ListView,
+  RefreshControl,
+} from 'react-native';
+import InfiniteScrollView from 'react-native-infinite-scroll-view';
+
+class ExampleComponent extends React.Component {
+  static propTypes = {
```

Comment:
Done - I hope this is the right syntax - is there a PropType defined by react-redux I should use instead?